lineapy/api/api.py

```diff
@@ -373,6 +373,7 @@ def to_pipeline(
     reuse_pre_computed_artifacts: List[str] = [],
     generate_test: bool = False,
     pipeline_dag_config: Optional[AirflowDagConfig] = {},
+    include_non_slice_as_comment: bool = False,
```

Comment:
These were gone after andrew merged yesterday. Maybe a rebase will give you cleaner diff.